rubocop.yml

```diff
@@ -1,7 +1,6 @@
 AllCops:
   Exclude:
   - 'db/schema.rb'
-  DisabledByDefault: true
```

Comment:
If we're removing this, we probably want to switch to explicitly disabling all existing rules that we haven't already got an entry for.

That way, we're only changing the behaviour going forward, instead of mass enabling rules on consumer repos.

Basically what you've done for `Gemspec/RequiredRubyVersion`, but we need to cover all the other rules which we might not detect _this repo_ violating. Not sure if there's an easy way to list out all those rules though.

Reply:
This feels overly paranoid to me. It makes more sense to bump the major version (`2.0`) and say “breaking change, watch out” with tips on how to deal with it. I don't think blanket trying to hide 10's or 100's of Cops is the answer and defeats the purpose of this change.

It should be up to each project which cops to ignore or fix, just as what was done in this PR.

Reply:
Can we test this out on a substantial project before release, to better understand what the upgrade process will involve?

And also maybe add an upgrade guide in the wiki here?